samples/dashboards/demo/climate/demo.js

```diff
@@ -390,7 +390,8 @@ async function setupBoard() {
                 },
                 tooltip: {
                     enabled: true,
-                    positioner: function (_width, _height, axisInfo) {
+                    distance: -60,
```

Comment:
Shouldn't be part of this commit ;) 
But if you would like to leave it, remove the positioner.